packages/react/src/styles.ts

```diff
@@ -0,0 +1 @@
+import './styles.css';
```

Comment:
OMG this is way cleaner than writing a build script to copy the file over. Bravo.